download.md

```diff
@@ -365,6 +365,13 @@ To improve the speed and load time of your site, build Prebid.js for only the he
     </div>
   </div>
 
+  <div class="col-md-4">
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" bidderCode="hiromedia" class="bidder-check-box"> HIRO Media
```

Comment:
can you add a disabled property here? We won't actually update the backend to support new bidders until they are in a tagged release. Thanks.